astropy/units/utils.py

```diff
@@ -202,7 +202,7 @@ def validate_power(p, support_tuples=False):
     p : float, int, Rational, Fraction
         Power to be converted
     """
-    if isinstance(p, (numbers.Rational, Fraction)):
+    if isinstance(p, (Fraction, numbers.Rational)):
```

Comment:
We use `Fraction` internally, and `numbers.Rational` is an ABC of which `Fraction` is a member - but `isinstance` on it is much slower. Might be even better to just remove it...